ramls/anonymize-loans-response.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Response schema for anonymize loans request",
+  "type": "object",
+  "properties": {
+    "anonimizedLoans": {
+      "description": "Successfully anonymized loan ids",
+      "type": "array",
+      "items":{
+        "type": "string"
+      }
+    },
+    "notAnonimizedLoans": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "$ref": "anonymize-loans-failure.json"
```

Comment:
Why has this been extracted as a separate schema?

Reply:
I guess that was the common practice in Folio. Shout it be inlined?

Reply:
This has been inlined.